src/components/ui/SuggestItem.jsx

```diff
@@ -1,56 +1,70 @@
 /* global _ */
 import React from 'react';
 import NavigatorGeolocalisationPoi from 'src/adapters/poi/specials/navigator_geolocalisation_poi';
-import IconManager from '../../adapters/icon_manager';
 import Category from 'src/adapters/category';
 import Intention from 'src/adapters/intention';
 import Address from 'src/components/ui/Address';
+import PlaceIcon from 'src/components/PlaceIcon';
+import { Magnifier } from 'src/components/ui/icons';
+import PoiStore from 'src/adapters/poi/poi_store';
 
 const ItemLabels = ({ firstLabel, secondLabel }) =>
   <div className="autocomplete_suggestion__labels">
     <div className="autocomplete_suggestion__first_line">{firstLabel}</div>
     {secondLabel && <div className="autocomplete_suggestion__second_line">{secondLabel}</div>}
   </div>;
 
-const GeolocationItem = () =>
+const GeolocationItem = ({ geolocPoi }) =>
   <div className="autocomplete_suggestion autocomplete_suggestion--geoloc">
-    <div className="autocomplete-icon icon-pin_geoloc" />
+    <PlaceIcon className="autocomplete_suggestion_icon" place={geolocPoi} withBackground />
     <ItemLabels firstLabel={_('Your position', 'direction')} />
   </div>;
 
 const IntentionItem = ({ intention }) => {
   const { category, place, fullTextQuery } = intention;
   const placeString = place
     ? `${_('Close to')} ${place.properties.geocoding.name}`
-    : _('Search around this place');
+    : _('nearby');
 
   return <div className="autocomplete_suggestion autocomplete_suggestion--intention">
-    <div className="autocomplete-icon" />
+    {category && <PlaceIcon
+      className="autocomplete_suggestion_icon"
+      iconClass={category.iconName}
```

Comment:
What do you think about passing the category itself to `PlaceIcon` instead of specifying `iconClass` and `color` as separate props?